packages/paste-core/components/alert/src/index.tsx

```diff
@@ -0,0 +1,126 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import {ValueOf} from '@twilio-paste/types';
+import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+import {MediaObject, MediaFigure, MediaBody} from '@twilio-paste/media-object';
+import {Button} from '@twilio-paste/button';
+import {CloseIcon} from '@twilio-paste/icons/esm/CloseIcon';
+import {ErrorIcon} from '@twilio-paste/icons/esm/ErrorIcon';
+import {InformationIcon} from '@twilio-paste/icons/esm/InformationIcon';
+import {WarningIcon} from '@twilio-paste/icons/esm/WarningIcon';
+
+export const ROLES = {
```

Comment:
mega nit: these constants should be PascalCase and the Keys all UpperCased for consistency (paste-website/src/constants.ts)